spec/spec_helper.rb

```diff
@@ -3,6 +3,7 @@
 Capybara.javascript_driver = :poltergeist
 
 RSpec.configure do |config|
+  Dir["./spec/support/**/*.rb"].sort.each { |f| require f }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.